src/shifts.jl

```diff
@@ -13,15 +15,15 @@ function within(hour, shift)::Bool
     end
 end
 
-function get_shifts_distance(first_shift, second_shift)
+function get_distance(first_shift::ShiftType, second_shift::ShiftType)::Int
```

Comment:
Czuję, że coś tu jest nie tak, koniecznie unittesty na tę funkcję, trochę sobie przetestowałem, jeśli coś nie czaję to popraw moje rozumowanie:
```julia
julia> function get_distance(x1, x2)
           if x1["from"] < x1["to"]
              24 + x2["from"] - x1["to"]
           else
              x2["from"] - x1["to"]
           end
       end

julia> x
Dict{String,Int64} with 2 entries:
  "to"   => 15
  "from" => 12

julia> y
Dict{String,Int64} with 2 entries:
  "to"   => 20
  "from" => 16

julia> get_distance(x, x)
21  # 0 albo 24?

julia> get_distance(y, y)
20  # 0 albo 24?

julia> get_distance(x, y)
25  # 1?

julia> get_distance(y, x)
16  # tu jest ok

julia> y = Dict("from" => 20, "to" => 4)
Dict{String,Int64} with 2 entries:
  "to"   => 4
  "from" => 20

julia> get_distance(x, y)
29  # 6?

julia> get_distance(y, x)
8  # tu jest ok
```

Reply:
Okay, może trochę źle udokumentowałem tą funkcję, moja wina xD. Ogólnie jej zamysł jest taki, że zwraca czas w godzinach, pomiędzy początkiem drugiej zmiany, a końcem pierwszej, przy założeniu, że druga zmiana jest wpisana dzień po pierwszej.

Reply:
Po przeliczeniu na kartce wydaje mi się, że wszystkie przypadki wyżej opisane, zgadzają się z założeniami funkcji.